packages/exports/src/exportSlidesFullLesson.ts

```diff
@@ -20,16 +22,9 @@ export const exportSlidesFullLesson = async ({
   onStateChange: (state: State<OutputData>) => void;
 }): Promise<Result<OutputData>> => {
   try {
-    const speakerNotesTagsToDelete: SpeakerNotesTag[] = [];
-    if (!lesson.cycle1) {
-      speakerNotesTagsToDelete.push("cycle1");
-    }
-    if (!lesson.cycle2) {
-      speakerNotesTagsToDelete.push("cycle2");
-    }
-    if (!lesson.cycle3) {
-      speakerNotesTagsToDelete.push("cycle3");
-    }
+    const speakerNotesTagsToDelete: SpeakerNotesTag[] = [1, 2, 3]
+      .filter((num) => !lesson[`cycle${num}` as keyof typeof lesson])
+      .map((num) => `cycle${num}` as SpeakerNotesTag);
```

Comment:
Same question as before around assertion, feels like we're losing a bit of typesafety here vs the previous implementation

Reply:
Same - because it's strong manipulation it complains without the assertion